test/meta.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "upload-ci-test",
+  "module_id": "e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb:upload-ci-test",
```

Comment:
why uuid instead of namespace here? for testing?

Reply:
No real reason. I guess namespace is prettier. Switched.